docs/manual/layout/templates/template-assets.en.md

```diff
@@ -33,3 +33,10 @@ $GLOBALS['TL_JAVASCRIPT'][] = 'files/myfolder/custom.js|static';
 Doing it this way, there are more options: By adding `|static`, for example, the files will be appended to or combined 
 with existing assets from page layouts. A detailed description of all options and output locations can be found in the
 developer documentation under [Adding CSS &amp; JavaScript Assets](https://docs.contao.org/dev/framework/asset-management/).
+
+
+#### Twig template support
+
+{{< version "4.13" >}}
+
+When using Twig templates, the options for combined inclusion in the HTML header are not available.
```

Comment:
```suggestion
When using Twig templates, the options for combined inclusion in the HTML header are not available.
```

@m-vo don't we have a Twig extension to allow execution of arbitrary PHP code in Twig templates?

Reply:
Lol, no? :smile: 

But we should IMHO not document this (in this way). There is already an open issue to get something implemented in this regards. We should find a solution there first before we tell users how to use it or what is possible and what not.

Reply:
Then I remembered it wrong ;).

And yes I agree.

Reply:
Agree, and mentioned it in the description regarding [3834](https://github.com/contao/contao/issues/3834)

Reply:
Changed or prepared for update with the last commits